api/swagger.yaml

```diff
@@ -5904,6 +5904,8 @@ paths:
           description: "The volume was created successfully"
           schema:
             $ref: "#/definitions/Volume"
+        304:
+          description: "The volume has already been created"
```

Comment:
Just mention that, there are another two 304 status codes in swagger.yml where we defined the schema as Error. While I think maybe we should take that into consideration as well.